components/GetHistogram.coffee

```diff
@@ -92,7 +91,7 @@ exports.getComponent = ->
       y: zero new Array 256
       h: zero new Array 361 # degrees [0,0, 360.0] -> [0, 361]
       s: zero new Array 101 # [0.0, 1.0] -> [0, 101]
-      c: zero new Array 101 # ?
+      c: zero new Array 135 # ?
```

Comment:
Because it was not instantiated the right amount of dimensions before (https://github.com/gka/chroma.js/issues/63).